MIGRATE_TO_KNAPSACK_PRO.md

```diff
@@ -0,0 +1,77 @@
+# Migration steps: from Knapsack to Knapsack Pro
+
+Follow these steps to migrate from `knapsack` to `knapsack_pro` in 10 minutes.
+
+Commands are provided to help you with each step.
+
+On Linux, you need to remove the `''` part from the `sed` commands. Also, you can ignore the `sed: no input files` warning that is printed when there's no substitution to perform.
```

Comment:
```suggestion
> [!TIP]
> On Linux, you need to remove the `''` part from the `sed` commands. Also, you can ignore the `sed: no input files` warning that is printed when there's no substitution to perform.
```

to get a tip look:
![image](https://github.com/KnapsackPro/knapsack/assets/662545/91bbc505-ee5a-4740-932f-abd074e01ba9)

source: https://stackoverflow.com/a/72327818/905697

Reply:
Oh. I didn't know about that!